com.pojtinger.felicitas.Multiplex.json

```diff
@@ -0,0 +1,648 @@
+{
+  "app-id": "com.pojtinger.felicitas.Multiplex",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "44",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.golang"],
+  "command": "multiplex",
+  "finish-args": [
+    "--share=network",
+    "--socket=fallback-x11",
```

Comment:
--share=ipc is necessary (see the screenshot you posted)

Reply:
Oh, I see - so we need `--share=ipc` in order to use spawn? I tried running it with this manifest, but it seemed to be able to launch IPC w/o issues since the Flatpak DBus talk is allowlisted.

Reply:
not. It's for the line quoted above. ^^^^^^^^

![image](https://github.com/flathub/flathub/assets/114441/3a8c6ba8-6ae3-424e-83a0-2c70b34642e0)

Reply:
Oh, thanks, now I see! That makes more sense. See ea4adc79ecb01ca57816b45ac5f1950c4e917228